### DISCUSSION TITLE
What is the smallest organ in the human body?

### DISCUSSION THREAD

**Discussion by SANKARAMDAS** [ParanoidUser/yolo#712] [Galaxy Trivia] [answered]
*don't forget to mention the location.

**Comment by ParanoidUser** [comment, marked as answer]
The **pineal gland** is a tiny endocrine gland in the center of the brain that helps regulate your body's circadian rhythm by secreting the hormone melatonin. The name *pineal* comes as pineal is a small pine-shaped gland.
![image](https://github.com/ParanoidUser/yolo/assets/5120290/e2aff449-59a3-4666-97f9-7330403d23ec)
